system.go

```diff
@@ -69,3 +70,19 @@ func (s *SystemService) GetDiagnosticID() string {
 	// TODO: Consider deprecating/rewriting in favor of just using GetUnsanitizedConfig().
 	return s.api.GetDiagnosticId()
 }
+
+// RequestTrialLicense requests a trial license and installs it in the server.
+// If the server version is lower than 5.36.0, an error is returned.
+//
+// Minimum server version: 5.4
```

Comment:
I set this to the minimum server version of GetServerVersion, which is the only API call we do without actually checking the server version before.

Reply:
Is that useful, though? Seems like having a floor on which we can tell if an error occurred isn't what the consumer of this API might want, but when the API is likely to be successful.

Reply:
To be clear: You're suggesting we require 5.36.0 and don't worry about the manual version test?

Reply:
No, I think it's fine to have the manual test and return an explicit error to the client that this API isn't supported. In fact, I'd love for all the plugin API wrappers to effectively allow this, making it easy to test distinguish downstream.

I'm just objecting to the documentation showing this as v5.4:
* From v5.2-v5.4, this API will fail cryptically
* From v5.4-v5.35, this API will fail with a hard-coded error
* From v5.36, this API will return the desired result.

I /feel/ like v5.36 is the version we're intending to communicate here, not that we successfully avoid a cryptic error.